cfme/utils/appliance/__init__.py

```diff
@@ -522,13 +528,14 @@ def configure(self, log_callback=None, **kwargs):
 
             # Debugging - ifcfg-eth0 overwritten by unknown process
             # Rules are permanent and will be reloade after machine reboot
-            self.ssh_client.run_command(
-                "cp -pr /etc/sysconfig/network-scripts/ifcfg-eth0 /var/tmp", ensure_host=True)
-            self.ssh_client.run_command(
-                "echo '-w /etc/sysconfig/network-scripts/ifcfg-eth0 -p wa' >> "
-                "/etc/audit/rules.d/audit.rules", ensure_host=True)
-            self.ssh_client.run_command("systemctl daemon-reload", ensure_host=True)
-            self.ssh_client.run_command("service auditd restart", ensure_host=True)
+            with self.ssh_client as ssh_client:
+                ssh_client.run_command(
+                    "cp -pr /etc/sysconfig/network-scripts/ifcfg-eth0 /var/tmp", ensure_host=True)
+                ssh_client.run_command(
+                    "echo '-w /etc/sysconfig/network-scripts/ifcfg-eth0 -p wa' >> "
+                    "/etc/audit/rules.d/audit.rules", ensure_host=True)
+                self.httpd.daemon_reload()
+                self.auditd.restart()
```

Comment:
auditd can't be start/stop via systemctl, RH directs users to use service command for auditd restart.